Doc/faq/programming.rst

```diff
@@ -419,7 +419,7 @@ What is the difference between arguments and parameters?
 
 :term:`Parameters <parameter>` are defined by the names that appear in a
 function definition, whereas :term:`arguments <argument>` are the values
-actually passed to a function when calling it.  Parameters define what types of
+actually passed to a function when calling it.  Parameters define what kind of
```

Comment:
Maybe we could make `kind of arguments` a link to https://docs.python.org/3/glossary.html#term-parameter as novice users may not know what it is exactly.

Reply:
Hi. Since I signed CLA today, it may take some time for bots to "know". (I received the PDF doc though) ^^.
I agree with your idea of linking `kind of arguments`, you may close the issue and do it from your side or wait for my CLA to get known. Either way, I'm happy.

Reply:
There is no issue to wait so the bot can find your CLA :)

Reply:
Hi. Another day has passed and the bot didn't find my CLA.
But I had received the certificate, what's going wrong then?

Reply:
Are you sure you correctly set your GitHub name on your bpo profile (https://cloud.githubusercontent.com/assets/2680980/23276970/d14a380c-f9d1-11e6-883d-e13b6b211239.png) ?

Reply:
Thanks for the follow up. I think "Github Name" refers to username (and not profile name). In that case I had set it as `j-arun-mani` but Github shown my username as `J-Arun-Mani`. I don't know if username-id is case sensitive, however, I edited my bpo profile to include the proper case. :)

Reply:
Thanks! It's now good :)

Reply:
Please make `kind of arguments` a link, also I had not noticed but the PR should be against `python:master`, not `python:3.8`. You should by able to change this by using the "Edit" button, no need to open another PR.